src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,156 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+import telemetry 0.30
+
+VPNTabNavigation {
+    property string _menuTitle: VPNl18n.TipsAndTricksSettingsEntryLabel
+    objectName: "settingsTipsAndTricksPage"
+
+    tabList: ListModel {
+        id: tabButtonList
+        ListElement {
+            tabLabelStringId: "TipsAndTricksTabTitleAll"
+            tabButtonId: "tabAll"
+        }
+        ListElement {
+            tabLabelStringId:"TipsAndTricksTabTitleTutorials"
+            tabButtonId: "tabTutorials"
+        }
+        ListElement {
+            tabLabelStringId:"TipsAndTricksTabTitleTips"
+            tabButtonId: "tabTips"
+        }
+    }
+
+    stackContent: [
+        // All
+        VPNFlickable {
+            flickContentHeight: layoutAll.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+            interactive: flickContentHeight > height
+
+            ObjectModel {
+                id: tipsAndTricksSections
+
+                Item { 
+                    property string title: VPNl18n.TipsAndTricksQuickTipsTitle
+                    property string description: VPNl18n.TipsAndTricksQuickTipsDescription
+                    property var filter: (addon) => !addon.advanced && !addon.highlighted
+                }
+
+                Item { 
+                    property string title: VPNl18n.TipsAndTricksAdvancedTipsTitle
+                    property string description: VPNl18n.TipsAndTricksAdvancedTipsDescription
+                    property var filter: (addon) => addon.advanced && !addon.highlighted
+                }
+            }
+
+            ColumnLayout {
+                id: layoutAll
+
+                anchors.fill: parent
+                anchors.topMargin: VPNTheme.theme.vSpacing
+                anchors.bottomMargin: VPNTheme.theme.vSpacing
+                anchors.leftMargin: VPNTheme.theme.windowMargin
+                anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                VPNTutorialList {
+                    parentWidth: layoutAll.width
+                    customTutorialFilter: (addon) => addon.highlighted
+                }
+
+                Repeater {
+                    model: tipsAndTricksSections.count
+
+                    Column {
+                        property var section: tipsAndTricksSections.get(index)
+
+                        visible: guidesList.count || tutorialsList.count
+
+                        Layout.alignment: Qt.AlignTop
+                        Layout.topMargin: 32
+                        spacing: VPNTheme.theme.vSpacingSmall
+
+                        VPNBoldLabel {
+                            text: section.title
+
+                            Accessible.role: Accessible.StaticText
+                            Accessible.name: text
+                        }
```

Comment:
I believe this needs horizontal anchors and wrapping (via `wrapMode`), otherwise long text will break the layout

Reply:
Oh yeah, I'm sorry I think that was here already, but I had such a massive fight with QML to get my widths and heights straight and removed so much stuff, some of it I must have forgotten to bring back 🤦‍♀️